app/services/articles/analytics_updater.rb

```diff
@@ -0,0 +1,57 @@
+module Articles
+  class AnalyticsUpdater
+    def initialize(context = "default")
+      @context = context
+    end
+
+    def self.call(*args)
+      new(*args).call
+    end
+
+    def call(user)
```

Comment:
This should be argument less, you can see the structure of other services like https://github.com/thepracticaldev/dev.to/blob/master/app/services/notifications/milestone/send.rb

Reply:
You are right, I have updated.